modules/preview-react/text/lib/Text.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import {createComponent} from '@workday/canvas-kit-react/common';
+// type changes no need
+import {type, CanvasTypeHierarchy, CanvasTypeVariants} from '@workday/canvas-kit-react/tokens';
+import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+
+export interface TextProps extends BoxProps {
+  /**
+   * Type token level: `title`, `heading`, `body`, `subtext`. Should be provided with `size` prop or it will not apply token.
+   */
+  level?: keyof CanvasTypeHierarchy;
+  /**
+   * Type token size: `large`, `medium`, `small`. Should be provided with `level` prop or it will not apply token.
+   */
+  size?: 'large' | 'medium' | 'small';
```

Comment:
can you explain a little bit more why we need a number?

Reply:
Hmm, i was thinking of `fontSize` so now we have a `size` prop and the consumer can still pass a `fontSize` prop right? Is there any way we can consolidate this? I'm referencing this as an example: https://chakra-ui.com/docs/components/text#changing-the-font-size

I'm just thinking out loud, keeping this as is is fine but something to think about

Reply:
so `size` is different from `fontSize`. `size` is just level token size. Maybe it's better to rename it to `levelSize` or make a one prop for level and size to pass a string like `heading.small`. But with the second way `size` prop will be still required in type level components 🤔

Reply:
i think `levelSize` reads better because i still have `fonSize` if I want to make it bigger right? It would be cool if we can extend `fontSize` style property to accept our levels PLUS whatever size you want:
 `fontSize="large"`
 `fontSize={16}`
 `fontSize="3em"`

Reply:
we already have this possibility as a Box prop.